extensions/default/src/DicomWebDataSource/qido.js

```diff
@@ -123,9 +124,9 @@ async function search(
 export function seriesInStudy(dicomWebClient, studyInstanceUID) {
   // Series Description
   // Already included?
-  const commaSeparatedFields = ['0008103E'].join(',');
```

Comment:
Do not replace '0008103E' with '00080021'. We want them both. Just make it `['0008103E', '00080021'].join(',')`

Reply:
Nevermind, fixed via Github UI